exts/mod.py

```diff
@@ -1,9 +1,10 @@
 import asyncio
 import importlib
+import re
 import time
+from contextlib import suppress
 from datetime import datetime, timedelta
 
-import naff
```

Comment:
Why is this being removed?

Reply:
Polls I beg look like 5 lines below this one and you'll see its usage—

Reply:
![image](https://user-images.githubusercontent.com/41456914/218335319-6725e7ad-3871-4996-ab80-9c49dd0bad0a.png)

Reply:
Fun fact. No need to beg. I use it myself 😅

Auto optimise imports wanted it gone. I saw it was in const so assumed I added it and you were doing some weird monkey patching